### DISCUSSION TITLE
Hierarchical time series forecasting

### DISCUSSION THREAD

**Discussion by cdeterman** [online-ml/river#740] [Q&A] [answered]
Does river support any sort of hierachical time series forecasting?  For example, take an economic index that as a fine granularity down to say the zip code level.  This has a natural hierarchy up through counties, states, regions.  I am familiar with typical time series methods such as the simple top-down or bottom-up approaches but curious if anything exists within river.

**Comment by MaxHalford** [comment, marked as answer]
Nope it doesn't! :)
